clippy_lints/src/literal_representation.rs

```diff
@@ -26,6 +26,25 @@ declare_clippy_lint! {
     "long integer literal without underscores"
 }
 
+/// **What it does:** Warns for mistyped suffix in literals
+///
+/// **Why is this bad?** This is most probably a typo
+///
+/// **Known problems:**
+///		- Recommends unsigned suffix even if it's a negative number. Should only  recommend the signed suffix on those cases.
```

Comment:
That is not true anymore. Maybe "always suggests singed integer", but I don't think this is really a problem.